cmd/destroy/destroy.go

```diff
@@ -236,6 +236,28 @@ func getTempKubeConfigFile(name string) string {
 	return filepath.Join(config.GetOktetoHome(), tempKubeconfigFileName)
 }
 
+func shouldRunInRemote(opts *Options) bool {
+	ld := &localDestroyCommand{}
+
+	// already in remote so we need to deploy locally
+	if utils.LoadBoolean(constants.OktetoDeployRemote) {
+		return false
+	}
+
+	// --remote flag enabled from command line
+	if opts.RunInRemote {
+		return true
+	}
+
+	//  remote option set in the manifest via a remote deployer image or the remote option enabled
+	if ld.manifest != nil && ld.manifest.Destroy != nil {
```

Comment:
we need to read manifest from `opt` otherwise, if we read it from `localDestroyCommand` struct properties , it will be `nil` always. We need to follow same approach than [in deploy command ](https://github.com/nitishfy/okteto/blob/Nitish/okteto-pipeline-flag/cmd/deploy/deploy.go#L514) but using destroy section when checking remotes options in manifest